src/request/src/middleware/src/storage/index.js

```diff
@@ -127,8 +127,8 @@ export default class Storage {
 
   removeById(collection, id) {
     return queue.add(() => {
-      if (!isString(id)) {
-        return Promise.reject(new Error('id must be a string', id));
+      if (isString(id) === false) {
```

Comment:
Same as answer above. I will adopt the shorthand for booleans and start with this code.